src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -111,6 +125,7 @@ def get_folder_service(s, name):
             self.config,
             self.service,
             lambda: self.config.api_token,
+            create_tahoe_client(DecodedURL.from_text(u""), StubTreq(Resource())),
```

Comment:
Maybe have a comment on why `StubTreq` with an empty resource is sufficient here?